src/date_extraction.py

```diff
@@ -29,9 +31,4 @@ def extract_date_from_str(patterns_list, string):
                 return parse(match)
             except:
                 print(f"String {match} was not recognised as date. Check date patterns")
-                return False
-
-
-if __name__ == '__main__':
-    string = 'letter_of_resignation_2011/01/01 08:15:27.123456.xlsx'
-    date = extract_date_from_str(patterns_list=patterns_list, string=string)
+                return False
```

Comment:
Probably makes more sense to return `None` than `False`?

Reply:
Agree, have changed that as well